### PR TITLE
Complete navigation website

### PR DESCRIPTION
**Pull request by Bettenoch**
In this branch we made the following changes: 

- Add Navigation bar with three sections book-list,  add page,  and contact-information page ✔.
- Add on click properties to the links to display only one of the three content sections ✔.

Thank You for reviewing our repository!!🙏

### CODE REVIEW

**Review by 1GORDON — COMMENTED** [review]
##
## Hi @Mike47ip, and @Bettenoch ✋

Good job so far!

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

### This project will be marked as Invalid and won't affect the number of your weekly reviews.

- [x] Kindly remove the live demo section from your readme file since this project is not deployed. This is a general requirement for all projects.

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@1GORDON` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.js [hunk 1]
```diff
@@ -1,73 +1,111 @@
...
+});
+
+addLink.addEventListener('click', () => {
+  addLink.classList.toggle('clicked2');
+  mainLink.classList.remove('clicked');
+  contactLink.classList.remove('clicked3');
+  awesomeBooks.classList.remove('more');
+  contactPage.classList.remove('add');
+  formPage.classList.remove('less');
+});
```
Comment (frempongdev) [inline index.js]: Good use of `add` and `remove` `classList` methods of hiding and displaying your page items. 🚀 However there is one tiny change I would suggest you make to complete your application. ♻️ 

- [ ] Kindly make sure that all page elements appear after their corresponding navigation link is clicked. 🖱️ Currently, your `Add page` link does not display the `Add Book` form after the `Contact` link has been clicked. 

### Please check as shown in the GIF below ⬇️ 
missing book form

I suggest that you remove that `class` that hides the form when the `Contact` link is clicked. Consider the code below ⬇️ 
  `formPage.classList.remove('less', 'remove');`

**Review by frempongdev — CHANGES_REQUESTED** [review]
# STATUS: CHANGES REQUIRED ♻️ 🚧 

Hi @Team  👋🏼 
I'm Frempong.

You've done a great job implementing most of the requirements for this project 🚀, but there are a few changes I recommend be made. You're almost done, keep up the good work! 🤖 🎆 



 ## Highlights
✅ No Linter errors. 🆗  
✅ Great implementation of logic. ✔️ 
✅ HTML, CSS & JavaScript best practices. 
💹 Good Looking UI


 ## Required Changes ♻️
_Kindly Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by usorfaitheloho — APPROVED** [review]
Hi @Mike47ip  , 
# STATUS: APPROVED! 🥇 💯 
- [x] Very Nice Design, Keep it up!!
done

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**